src/czbenchmarks/utils.py

```diff
@@ -115,15 +127,10 @@ def load_custom_config(
 
         # Disable strict checking and merge configs
         # TODO check if custom_cfg introduces new keys and only disable strict checking if needed
```

Comment:
-TODO now?

Reply:
Actually, TODO is not addressed, but could be moved to `_disable_strict_checking_if_required` now perhaps.